### ISSUE TITLE
Installation via "jpm install https://github..." doesn't work

### ISSUE THREAD

**Issue by magv** [janet-lang/jaylib#12]
Hi, folks. With the latest janet/jpm (1.12.1-15e05b69) on Ubuntu 18.04:
```
$ env JANET_MODPATH=$PWD/modpath jpm install https://github.com/janet-lang/jaylib.git
...
raylib/src/core.c:191:14: fatal error: GLFW/glfw3.h: No such file or directory
     #include <GLFW/glfw3.h>         // GLFW3 library: Windows, OpenGL context and Input management
              ^~~~~~~~~~~~~~
```

It is my understanding that raylib comes with glfw bundled (`raylib/src/external/glfw`), so this means some paths are not configured correctly for some reason. Any ideas?

**Comment by magv** [comment]
Same problem with just `jpm build` after checkout.

This fixes the build for me:
```diff
diff --git a/project.janet b/project.janet
index b2382ec..2e96d27 100644
--- a/project.janet
+++ b/project.janet
@@ -8,7 +8,7 @@
   (case o
     :macos '["-Iraylib/src" "-ObjC"]
     #default
-    '["-Iraylib/src"]))
+    '["-Iraylib/src" "-Iraylib/src/external/glfw/include" "-Iraylib/src/external/glfw/deps/mingw"]))
 
 (def lflags
   (case o
```

**Comment by yumaikas** [comment]
I just tested this, and I didn't have to include MINGW, just glfw

**Comment by bakpakin** [comment]
Merged #13 , closing.
